ui/admin/tests/acceptance/groups/selectors.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const MANAGE_DROPDOWN_SELECTOR =
+  '[data-test-manage-group-dropdown] button:first-child';
+export const DELETE_ACTION_SELECTOR =
+  '[data-test-manage-group-dropdown] ul li button';
```

Comment:
small suggestion: with the new naming conventions im noticing in other files, we don't need to add `SELECTOR` to the name. We could change this to `MANAGE_DROPDOWN` and `MANAGE_DROPDOWN_DELETE` to match what we have for auth-methods.

Reply:
thanks for the info, I updated!